pkg/tmpl/context_funcs_test.go

```diff
@@ -221,3 +223,32 @@ func TestRequired(t *testing.T) {
 		})
 	}
 }
+
+// TestRequiredEnv tests that RequiredEnv returns an error if the environment variable is not set.
+func TestRequiredEnv(t *testing.T) {
+
+	// test that the environment variable is not set
+	envKey := "HelmFile"
+	envVal, err := RequiredEnv(envKey)
+
+	require.NotNilf(t, err, "Expected error to be returned when environment variable %s is not set", envKey)
+	require.Emptyf(t, envVal, "Expected empty string to be returned when environment variable %s is not set", envKey)
```

Comment:
Nit but AFAIK any error stops template execution when it retuned an non-nil error it won't matter match what's returned when it errored! But it's also true there might be no bad side-effect having this covered in the test.